src/libultra/voice/voiceinit.c

```diff
@@ -6,15 +6,15 @@
 
 #include "ultra64/controller_voice.h"
 #include "ultra64/os_voice.h"
-#include "io/controller.h"
+#include "ultra64/controller.h"
 #include "functions.h"
 #include "macros.h"
 
 static u8 sCmds[] = {
     0x1E, 0x6E, 0x08, 0x56, 0x03,
 };
 
-s32 osVoiceInit(OSMesgQueue* mq, OSVoiceHandle* hd, s32 channel) {
+s32 osVoiceInit(OSMesgQueue* mq, OSVoiceHandle* hd, int channel) {
```

Comment:
Is there a reason this and the arg in `osVoiceMaskDictionary` are changed to `int`?

Reply:
To match what `ultralib` has, but I can undo it if preferred

Reply:
I don't feel that strongly about it, just seems odd to use both `s32` as the return but `int` as an arg